examples/semantic_segmentation/requirements.txt

```diff
@@ -0,0 +1,5 @@
+torch==1.13.1 
```

Comment:
Why does it have torch requirement while the one for image classification task doesn't?

Reply:
I added by mistake, let me remove it